docs/tex/secciones/01_introduccion.tex

```diff
@@ -0,0 +1,30 @@
+\chapter{Introducción}
+
+No se puede negar que internet es actualmente uno de los medios de comunicación más efectivos (si no el que más), ya que permite una comunicación rápida y de alcance global entre todos sus usuarios con una gran libertad.
+\\\\
+Para explotar al máximo estas características se han ido creando diversas plataformas para facilitar la comunicación entre usuarios, junto a las cuales han surgido diversas ideas sobre como explotar ese tráfico de usuarios y generar beneficios a la vez que se prestan servicios aparentemente gratuitos.
```

Comment:
Now that I read this, I noticed that it feels really out of place and... unrelated? Also since this is part of #1 it is not complete since it lacks the motivation, I'll have to add it to the PR as well...

Reply:
Just fixed this and added problem description and motivation. PR is even larger but I think this was required so the rest of it could make sense